website/data/slack/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
do we  need this file?
@erezrokah

Reply:
Yes we do, it's specific to the Slack plugin to make installation into a workspace with all the necessary permissions easier